Try/hint of more blocks in inserter

enriquesanchez opened the pull request:
## Description
Closes #17872.

This PR explores setting `line-height: 1;` to some button elements in the block inserter and adding `10px` to the inserter's overall height in order to reveal more of the available blocks and categories, thus hinting to users that there's more content to scroll.

## How has this been tested?
Locally in MacOS Firefox, Safari and Chrome.

## Screenshots <!-- if applicable -->

**Before the change:** 
<img width="715" alt="66523022-7f2a4400-eab4-11e9-940c-f22a183959b2" src="https://user-images.githubusercontent.com/3276087/72297126-d6decb00-3620-11ea-870a-020aa546be82.png">


**After the change:**
<img width="729" alt="Screen Shot 2020-01-13 at 16 18 35" src="https://user-images.githubusercontent.com/3276087/72297376-6b492d80-3621-11ea-836e-54cd80c2ff42.png">


## Types of changes
CSS, visual.

enriquesanchez committed:
Set buttons line-height to 1

enriquesanchez committed:
Slightly increase height of block inserter

karmatosed commented:
@enriquesanchez I was just testing this PR and I wonder if you could go a little further and show all of the accordion? For me, not showing it all felt a little disjointed.

enriquesanchez committed:
Add a bit more of height to show more of the accordion

enriquesanchez commented:
thanks for the feedback @karmatosed!

I updated the PR and added more height to the inserter, it should now look like this:

<img width="880" alt="Screen Shot 2020-01-21 at 18 25 23" src="https://user-images.githubusercontent.com/3276087/72854923-7c2a1c80-3c7b-11ea-9a60-5d950d5862c3.png">

draganescu commented:
@enriquesanchez I am unsure of my dev env lately but for me on this branch, unline master I cannot open the block inserter on mobile:

<img width="384" alt="Screenshot 2020-02-13 at 08 43 47" src="https://user-images.githubusercontent.com/107534/74408413-03af1980-4e3d-11ea-9fea-eb04491c93e3.png">

enriquesanchez commented:
Thanks for the feedback @karmatosed and @draganescu 🙏

Given that this is a tiny change and there's already other issues (#19836 and #17335) addressing changes to the block library, I'm going to go ahead and close this PR.
